
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://genomicsstandardsconsortium.github.io/mixs/WastewaterSludgeMIGSEukaryote/">
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.3.0, mkdocs-material-8.3.9">
    
    
      
        <title>WastewaterSludgeMIGSEukaryote - MIxS</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.1d29e8d0.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.cbb835fc.min.css">
        
      
      
    
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("..",location),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="" data-md-color-primary="none" data-md-color-accent="none">
  
    
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#wastewatersludgemigseukaryote" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="MIxS" class="md-header__button md-logo" aria-label="MIxS" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            MIxS
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              WastewaterSludgeMIGSEukaryote
            
          </span>
        </div>
      </div>
    </div>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" aria-label="Clear" tabindex="-1">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/GenomicsStandardsConsortium/mixs/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.1.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="MIxS" class="md-nav__button md-logo" aria-label="MIxS" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    MIxS
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/GenomicsStandardsConsortium/mixs/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.1.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        About
      </a>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_2" type="checkbox" id="__nav_2" >
      
      
      
      
        <label class="md-nav__link" for="__nav_2">
          Introduction
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Introduction" data-md-level="1">
        <label class="md-nav__title" for="__nav_2">
          <span class="md-nav__icon md-icon"></span>
          Introduction
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../introduction/background/" class="md-nav__link">
        Background
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../introduction/citation/" class="md-nav__link">
        Cite
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../introduction/contents/" class="md-nav__link">
        Contents
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#inheritance" class="md-nav__link">
    Inheritance
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slots" class="md-nav__link">
    Slots
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#usages" class="md-nav__link">
    Usages
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#identifier-and-mapping-information" class="md-nav__link">
    Identifier and Mapping Information
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          <div class="md-content" data-md-component="content">
            <article class="md-content__inner md-typeset">
              
                
  <a href="https://github.com/GenomicsStandardsConsortium/mixs/edit/master/docs/WastewaterSludgeMIGSEukaryote.md" title="Edit this page" class="md-content__button md-icon">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25Z"/></svg>
  </a>



<h1 id="wastewatersludgemigseukaryote">WastewaterSludgeMIGSEukaryote</h1>
<p>Combinatorial checklist Minimal Information about a Genome Sequence: eukaryote with environmental package wastewater_sludge</p>
<p>URI: <a href="https://w3id.org/mixs/vocab/WastewaterSludgeMIGSEukaryote">mixs.vocab:WastewaterSludgeMIGSEukaryote</a></p>
<h2 id="inheritance">Inheritance</h2>
<ul>
<li><a href="../WastewaterSludge/">WastewaterSludge</a><ul>
<li><strong>WastewaterSludgeMIGSEukaryote</strong> [ MIGS eukaryote]</li>
</ul>
</li>
</ul>
<!-- no inheritance hierarchy -->

<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>MIxS ID</th>
<th>Name</th>
<th>Multiplicity</th>
<th>Description</th>
<th>Pattern</th>
<th>Info</th>
</tr>
</thead>
<tbody>
<tr>
<td>MIXS:0001107</td>
<td><a href="../samp_name/">samp_name</a></td>
<td>1..1</td>
<td>A local identifier or name that for the material sample used for extracting nucleic acids, and subsequent sequencing. It can refer either to the original material collected or to any derived sub-samples. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible. INSDC requires every sample name from a single Submitter to be unique. Use of a globally unique identifier for the field source_mat_id is recommended in addition to sample_name.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001320</td>
<td><a href="../samp_taxon_id/">samp_taxon_id</a></td>
<td>1..1</td>
<td>NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa sample. Use 'synthetic metagenome’ for mock community/positive controls, or 'blank sample' for negative controls.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000092</td>
<td><a href="../project_name/">project_name</a></td>
<td>1..1</td>
<td>Name of the project within which the sequencing was organized</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000008</td>
<td><a href="../experimental_factor/">experimental_factor</a></td>
<td>0..1</td>
<td>Experimental factors are essentially the variable aspects of an experiment design which can be used to describe an experiment, or set of experiments, in an increasingly detailed manner. This field accepts ontology terms from Experimental Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI). For a browser of EFO (v 2.95) terms, please see http://purl.bioontology.org/ontology/EFO; for a browser of OBI (v 2018-02-12) terms please see http://purl.bioontology.org/ontology/OBI</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000009</td>
<td><a href="../lat_lon/">lat_lon</a></td>
<td>1..1</td>
<td>The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000018</td>
<td><a href="../depth/">depth</a></td>
<td>0..1</td>
<td>The vertical distance below local surface, e.g. for sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000094</td>
<td><a href="../alt/">alt</a></td>
<td>0..1</td>
<td>Altitude is a term used to identify heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth's surface. In this context, the altitude measurement is the vertical distance between the earth's surface above sea level and the sampled position in the air</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000093</td>
<td><a href="../elev/">elev</a></td>
<td>0..1</td>
<td>Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth's surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000113</td>
<td><a href="../temp/">temp</a></td>
<td>0..1</td>
<td>Temperature of the sample at the time of sampling.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000010</td>
<td><a href="../geo_loc_name/">geo_loc_name</a></td>
<td>1..1</td>
<td>The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (http://purl.bioontology.org/ontology/GAZ)</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000011</td>
<td><a href="../collection_date/">collection_date</a></td>
<td>1..1</td>
<td>The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001321</td>
<td><a href="../neg_cont_type/">neg_cont_type</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The substance or equipment used as a negative control in an investigation</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001322</td>
<td><a href="../pos_cont_type/">pos_cont_type</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The substance, mixture, product, or apparatus used to verify that a process which is part of an investigation delivers a true positive.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000012</td>
<td><a href="../env_broad_scale/">env_broad_scale</a></td>
<td>1..1</td>
<td>Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO’s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000013</td>
<td><a href="../env_local_scale/">env_local_scale</a></td>
<td>1..1</td>
<td>Report the entity or entities which are in the sample or specimen’s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000014</td>
<td><a href="../env_medium/">env_medium</a></td>
<td>1..1</td>
<td>Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of 'environmental material' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top).</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000020</td>
<td><a href="../subspecf_gen_lin/">subspecf_gen_lin</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Information about the genetic distinctness of the sequenced organism below the subspecies level, e.g., serovar, serotype, biotype, ecotype, or any relevant genetic typing schemes like Group I plasmid. Subspecies should not be recorded in this term, but in the NCBI taxonomy. Supply both the lineage name and the lineage rank separated by a colon, e.g., biovar:abc123.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000021</td>
<td><a href="../ploidy/">ploidy</a></td>
<td>0..1</td>
<td>The ploidy level of the genome (e.g. allopolyploid, haploid, diploid, triploid, tetraploid). It has implications for the downstream study of duplicated gene and regions of the genomes (and perhaps for difficulties in assembly). For terms, please select terms listed under class ploidy (PATO:001374) of Phenotypic Quality Ontology (PATO), and for a browser of PATO (v 2018-03-27) please refer to http://purl.bioontology.org/ontology/PATO</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000022</td>
<td><a href="../num_replicons/">num_replicons</a></td>
<td>0..1</td>
<td>Reports the number of replicons in a nuclear genome of eukaryotes, in the genome of a bacterium or archaea or the number of segments in a segmented virus. Always applied to the haploid chromosome count of a eukaryote</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000023</td>
<td><a href="../extrachrom_elements/">extrachrom_elements</a></td>
<td>0..1</td>
<td>Do plasmids exist of significant phenotypic consequence (e.g. ones that determine virulence or antibiotic resistance). Megaplasmids? Other plasmids (borrelia has 15+ plasmids)</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000024</td>
<td><a href="../estimated_size/">estimated_size</a></td>
<td>0..1</td>
<td>The estimated size of the genome prior to sequencing. Of particular importance in the sequencing of (eukaryotic) genome which could remain in draft form for a long or unspecified period.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000025</td>
<td><a href="../ref_biomaterial/">ref_biomaterial</a></td>
<td>0..1</td>
<td>Primary publication if isolated before genome publication; otherwise, primary genome report.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000026</td>
<td><a href="../source_mat_id/">source_mat_id</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A unique identifier assigned to a material sample (as defined by http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular digital record of a material sample) used for extracting nucleic acids, and subsequent sequencing. The identifier can refer either to the original material collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher, /bio_material, or /culture_collection may or may not share the same value as the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id may both contain 'UAM:Herps:14' , referring to both the specimen voucher and sampled tissue with the same identifier. However, the /culture_collection qualifier may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id would refer to an identifier from some derived culture from which the nucleic acids were extracted (e.g. xatc123 or ark:/2154/R2).</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000027</td>
<td><a href="../pathogenicity/">pathogenicity</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>To what is the entity pathogenic</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000028</td>
<td><a href="../biotic_relationship/">biotic_relationship</a></td>
<td>0..1</td>
<td>Description of relationship(s) between the subject organism and other organism(s) it is associated with. E.g., parasite on species X; mutualist with species Y. The target organism is the subject of the relationship, and the other organism(s) is the object</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000029</td>
<td><a href="../specific_host/">specific_host</a></td>
<td>0..1</td>
<td>Report the host's taxonomic name and/or NCBI taxonomy ID.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000030</td>
<td><a href="../host_spec_range/">host_spec_range</a></td>
<td>0..1</td>
<td>The range and diversity of host species that an organism is capable of infecting, defined by NCBI taxonomy identifier.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000031</td>
<td><a href="../host_disease_stat/">host_disease_stat</a></td>
<td>0..1</td>
<td>List of diseases with which the host has been diagnosed; can include multiple diagnoses. The value of the field depends on host; for humans the terms should be chosen from the DO (Human Disease Ontology) at https://www.disease-ontology.org, non-human host diseases are free text</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000032</td>
<td><a href="../trophic_level/">trophic_level</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Trophic levels are the feeding position in a food chain. Microbes can be a range of producers (e.g. chemolithotroph)</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000033</td>
<td><a href="../propagation/">propagation</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The type of reproduction from the parent stock. Values for this field is specific to different taxa. For phage or virus: lytic/lysogenic/temperate/obligately lytic. For plasmids: incompatibility group. For eukaryotes: sexual/asexual.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000034</td>
<td><a href="../encoded_traits/">encoded_traits</a></td>
<td>0..1</td>
<td>Should include key traits like antibiotic resistance or xenobiotic degradation phenotypes for plasmids, converting genes for phage</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000015</td>
<td><a href="../rel_to_oxygen/">rel_to_oxygen</a></td>
<td>0..1</td>
<td>Is this organism an aerobe, anaerobe? Please note that aerobic and anaerobic are valid descriptors for microbial environments</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000003</td>
<td><a href="../isol_growth_condt/">isol_growth_condt</a></td>
<td>1..1</td>
<td>Publication reference in the form of pubmed ID (pmid), digital object identifier (doi) or url for isolation and growth condition specifications of the organism/material</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000002</td>
<td><a href="../samp_collec_device/">samp_collec_device</a></td>
<td>0..1</td>
<td>The device used to collect an environmental sample. This field accepts terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO). This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094).</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001225</td>
<td><a href="../samp_collec_method/">samp_collec_method</a></td>
<td>0..1</td>
<td>The method employed for collecting the sample.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000016</td>
<td><a href="../samp_mat_process/">samp_mat_process</a></td>
<td>0..1</td>
<td>A brief description of any processing applied to the sample during or after retrieving the sample from environment, or a link to the relevant protocol(s) performed.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000017</td>
<td><a href="../size_frac/">size_frac</a></td>
<td>0..1</td>
<td>Filtering pore size used in sample preparation</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000001</td>
<td><a href="../samp_size/">samp_size</a></td>
<td>0..1</td>
<td>The total amount or size (volume (ml), mass (g) or area (m2) ) of sample collected.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000111</td>
<td><a href="../samp_vol_we_dna_ext/">samp_vol_we_dna_ext</a></td>
<td>0..1</td>
<td>Volume (ml) or mass (g) of total collected sample processed for DNA extraction. Note: total sample collected should be entered under the term Sample Size (MIXS:0000001).</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000035</td>
<td><a href="../source_uvig/">source_uvig</a></td>
<td>0..1</td>
<td>Type of dataset from which the UViG was obtained</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000036</td>
<td><a href="../virus_enrich_appr/">virus_enrich_appr</a></td>
<td>0..1</td>
<td>List of approaches used to enrich the sample for viruses, if any</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000037</td>
<td><a href="../nucl_acid_ext/">nucl_acid_ext</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the material separation to recover the nucleic acid fraction from a sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000038</td>
<td><a href="../nucl_acid_amp/">nucl_acid_amp</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the enzymatic amplification (PCR, TMA, NASBA) of specific nucleic acids</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000039</td>
<td><a href="../lib_size/">lib_size</a></td>
<td>0..1</td>
<td>Total number of clones in the library prepared for the project</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000040</td>
<td><a href="../lib_reads_seqd/">lib_reads_seqd</a></td>
<td>0..1</td>
<td>Total number of clones sequenced from the library</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000041</td>
<td><a href="../lib_layout/">lib_layout</a></td>
<td>0..1</td>
<td>Specify whether to expect single, paired, or other configuration of reads</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000042</td>
<td><a href="../lib_vector/">lib_vector</a></td>
<td>0..1</td>
<td>Cloning vector type(s) used in construction of libraries</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000043</td>
<td><a href="../lib_screen/">lib_screen</a></td>
<td>0..1</td>
<td>Specific enrichment or screening methods applied before and/or after creating libraries</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000044</td>
<td><a href="../target_gene/">target_gene</a></td>
<td>0..1</td>
<td>Targeted gene or locus name for marker gene studies</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000045</td>
<td><a href="../target_subfragment/">target_subfragment</a></td>
<td>0..1</td>
<td>Name of subfragment of a gene or locus. Important to e.g. identify special regions on marker genes like V6 on 16S rRNA</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000046</td>
<td><a href="../pcr_primers/">pcr_primers</a></td>
<td>0..1</td>
<td>PCR primers that were used to amplify the sequence of the targeted gene, locus or subfragment. This field should contain all the primers used for a single PCR reaction if multiple forward or reverse primers are present in a single PCR reaction. The primer sequence should be reported in uppercase letters</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000047</td>
<td><a href="../mid/">mid</a></td>
<td>0..1</td>
<td>Molecular barcodes, called Multiplex Identifiers (MIDs), that are used to specifically tag unique samples in a sequencing run. Sequence should be reported in uppercase letters</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000048</td>
<td><a href="../adapters/">adapters</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Adapters provide priming sequences for both amplification and sequencing of the sample-library fragments. Both adapters should be reported; in uppercase letters</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000049</td>
<td><a href="../pcr_cond/">pcr_cond</a></td>
<td>0..1</td>
<td>Description of reaction conditions and components of PCR in the form of 'initial denaturation:94degC_1.5min; annealing=...'</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000050</td>
<td><a href="../seq_meth/">seq_meth</a></td>
<td>1..1</td>
<td>Sequencing machine used. Where possible the term should be taken from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103).</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000051</td>
<td><a href="../seq_quality_check/">seq_quality_check</a></td>
<td>0..1</td>
<td>Indicate if the sequence has been called by automatic systems (none) or undergone a manual editing procedure (e.g. by inspecting the raw data or chromatograms). Applied only for sequences that are not submitted to SRA,ENA or DRA</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000052</td>
<td><a href="../chimera_check/">chimera_check</a></td>
<td>0..1</td>
<td>Tool(s) used for chimera checking, including version number and parameters, to discover and remove chimeric sequences. A chimeric sequence is comprised of two or more phylogenetically distinct parent sequences.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000053</td>
<td><a href="../tax_ident/">tax_ident</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The phylogenetic marker(s) used to assign an organism name to the SAG or MAG</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000056</td>
<td><a href="../assembly_qual/">assembly_qual</a></td>
<td>1..1</td>
<td>The assembly quality category is based on sets of criteria outlined for each assembly quality category. For MISAG/MIMAG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities with a consensus error rate equivalent to Q50 or better. High Quality Draft:Multiple fragments where gaps span repetitive regions. Presence of the 23S, 16S and 5S rRNA genes and at least 18 tRNAs. Medium Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Low Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Assembly statistics include, but are not limited to total assembly size, number of contigs, contig N50/L50, and maximum contig length. For MIUVIG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities, with extensive manual review and editing to annotate putative gene functions and transcriptional units. High-quality draft genome: One or multiple fragments, totaling ≥ 90% of the expected genome or replicon sequence or predicted complete. Genome fragment(s): One or multiple fragments, totalling &lt; 90% of the expected genome or replicon sequence, or for which no genome size could be estimated</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000057</td>
<td><a href="../assembly_name/">assembly_name</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Name/version of the assembly provided by the submitter that is used in the genome browsers and in the community</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000058</td>
<td><a href="../assembly_software/">assembly_software</a></td>
<td>1..1</td>
<td>Tool(s) used for assembly, including version number and parameters</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000059</td>
<td><a href="../annot/">annot</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Tool used for annotation, or for cases where annotation was provided by a community jamboree or model organism database rather than by a specific submitter</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000060</td>
<td><a href="../number_contig/">number_contig</a></td>
<td>1..1</td>
<td>Total number of contigs in the cleaned/submitted assembly that makes up a given genome, SAG, MAG, or UViG</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000061</td>
<td><a href="../feat_pred/">feat_pred</a></td>
<td>0..1</td>
<td>Method used to predict UViGs features such as ORFs, integration site, etc.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000062</td>
<td><a href="../ref_db/">ref_db</a></td>
<td>0..1</td>
<td>List of database(s) used for ORF annotation, along with version number and reference to website or publication</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000063</td>
<td><a href="../sim_search_meth/">sim_search_meth</a></td>
<td>0..1</td>
<td>Tool used to compare ORFs with database, along with version and cutoffs used</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000064</td>
<td><a href="../tax_class/">tax_class</a></td>
<td>0..1</td>
<td>Method used for taxonomic classification, along with reference database used, classification rank, and thresholds used to classify new genomes</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000065</td>
<td><a href="../x_16s_recover/">x_16s_recover</a></td>
<td>0..1</td>
<td>Can a 16S gene be recovered from the submitted SAG or MAG?</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000066</td>
<td><a href="../x_16s_recover_software/">x_16s_recover_software</a></td>
<td>0..1</td>
<td>Tools used for 16S rRNA gene extraction</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000067</td>
<td><a href="../trnas/">trnas</a></td>
<td>0..1</td>
<td>The total number of tRNAs identified from the SAG or MAG</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000068</td>
<td><a href="../trna_ext_software/">trna_ext_software</a></td>
<td>0..1</td>
<td>Tools used for tRNA identification</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000069</td>
<td><a href="../compl_score/">compl_score</a></td>
<td>0..1</td>
<td>Completeness score is typically based on either the fraction of markers found as compared to a database or the percent of a genome found as compared to a closely related reference genome. High Quality Draft: &gt;90%, Medium Quality Draft: &gt;50%, and Low Quality Draft: &lt; 50% should have the indicated completeness scores</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000070</td>
<td><a href="../compl_software/">compl_software</a></td>
<td>0..1</td>
<td>Tools used for completion estimate, i.e. checkm, anvi'o, busco</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000071</td>
<td><a href="../compl_appr/">compl_appr</a></td>
<td>0..1</td>
<td>The approach used to determine the completeness of a given genomic assembly, which would typically make use of a set of conserved marker genes or a closely related reference genome. For UViG completeness, include reference genome or group used, and contig feature suggesting a complete genome</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000072</td>
<td><a href="../contam_score/">contam_score</a></td>
<td>0..1</td>
<td>The contamination score is based on the fraction of single-copy genes that are observed more than once in a query genome. The following scores are acceptable for; High Quality Draft: &lt; 5%, Medium Quality Draft: &lt; 10%, Low Quality Draft: &lt; 10%. Contamination must be below 5% for a SAG or MAG to be deposited into any of the public databases</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000005</td>
<td><a href="../contam_screen_input/">contam_screen_input</a></td>
<td>0..1</td>
<td>The type of sequence data used as input</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000073</td>
<td><a href="../contam_screen_param/">contam_screen_param</a></td>
<td>0..1</td>
<td>Specific parameters used in the decontamination sofware, such as reference database, coverage, and kmers. Combinations of these parameters may also be used, i.e. kmer and coverage, or reference database and kmer</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000074</td>
<td><a href="../decontam_software/">decontam_software</a></td>
<td>0..1</td>
<td>Tool(s) used in contamination screening</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000075</td>
<td><a href="../sort_tech/">sort_tech</a></td>
<td>0..1</td>
<td>Method used to sort/isolate cells or particles of interest</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000076</td>
<td><a href="../single_cell_lysis_appr/">single_cell_lysis_appr</a></td>
<td>0..1</td>
<td>Method used to free DNA from interior of the cell(s) or particle(s)</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000054</td>
<td><a href="../single_cell_lysis_prot/">single_cell_lysis_prot</a></td>
<td>0..1</td>
<td>Name of the kit or standard protocol used for cell(s) or particle(s) lysis</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000055</td>
<td><a href="../wga_amp_appr/">wga_amp_appr</a></td>
<td>0..1</td>
<td>Method used to amplify genomic DNA in preparation for sequencing</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000006</td>
<td><a href="../wga_amp_kit/">wga_amp_kit</a></td>
<td>0..1</td>
<td>Kit used to amplify genomic DNA in preparation for sequencing</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000077</td>
<td><a href="../bin_param/">bin_param</a></td>
<td>0..1</td>
<td>The parameters that have been applied during the extraction of genomes from metagenomic datasets</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000078</td>
<td><a href="../bin_software/">bin_software</a></td>
<td>0..1</td>
<td>Tool(s) used for the extraction of genomes from metagenomic datasets, where possible include a product ID (PID) of the tool(s) used.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000079</td>
<td><a href="../reassembly_bin/">reassembly_bin</a></td>
<td>0..1</td>
<td>Has an assembly been performed on a genome bin extracted from a metagenomic assembly?</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000080</td>
<td><a href="../mag_cov_software/">mag_cov_software</a></td>
<td>0..1</td>
<td>Tool(s) used to determine the genome coverage if coverage is used as a binning parameter in the extraction of genomes from metagenomic datasets</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000081</td>
<td><a href="../vir_ident_software/">vir_ident_software</a></td>
<td>0..1</td>
<td>Tool(s) used for the identification of UViG as a viral genome, software or protocol name including version number, parameters, and cutoffs used</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000082</td>
<td><a href="../pred_genome_type/">pred_genome_type</a></td>
<td>0..1</td>
<td>Type of genome predicted for the UViG</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000083</td>
<td><a href="../pred_genome_struc/">pred_genome_struc</a></td>
<td>0..1</td>
<td>Expected structure of the viral genome</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000084</td>
<td><a href="../detec_type/">detec_type</a></td>
<td>0..1</td>
<td>Type of UViG detection</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000085</td>
<td><a href="../otu_class_appr/">otu_class_appr</a></td>
<td>0..1</td>
<td>Cutoffs and approach used when clustering “species-level” OTUs. Note that results from standard 95% ANI / 85% AF clustering should be provided alongside OTUS defined from another set of thresholds, even if the latter are the ones primarily used during the analysis</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000086</td>
<td><a href="../otu_seq_comp_appr/">otu_seq_comp_appr</a></td>
<td>0..1</td>
<td>Tool and thresholds used to compare sequences when computing "species-level" OTUs</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000087</td>
<td><a href="../otu_db/">otu_db</a></td>
<td>0..1</td>
<td>Reference database (i.e. sequences not generated as part of the current study) used to cluster new genomes in "species-level" OTUs, if any</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000088</td>
<td><a href="../host_pred_appr/">host_pred_appr</a></td>
<td>0..1</td>
<td>Tool or approach used for host prediction</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000089</td>
<td><a href="../host_pred_est_acc/">host_pred_est_acc</a></td>
<td>0..1</td>
<td>For each tool or approach used for host prediction, estimated false discovery rates should be included, either computed de novo or from the literature</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000091</td>
<td><a href="../associated_resource/">associated_resource</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A related resource that is referenced, cited, or otherwise associated to the sequence.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000090</td>
<td><a href="../sop/">sop</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Standard operating procedures used in assembly and/or annotation of genomes, metagenomes or environmental sequences</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000753</td>
<td><a href="../oxy_stat_samp/">oxy_stat_samp</a></td>
<td>0..1</td>
<td>Oxygenation status of sample.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000116</td>
<td><a href="../samp_store_dur/">samp_store_dur</a></td>
<td>0..1</td>
<td>Duration for which the sample was stored</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001001</td>
<td><a href="../ph/">ph</a></td>
<td>0..1</td>
<td>Ph measurement of the sample, or liquid portion of sample, or aqueous phase of the fluid</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000421</td>
<td><a href="../alkalinity/">alkalinity</a></td>
<td>0..1</td>
<td>Alkalinity, the ability of a solution to neutralize acids to the equivalence point of carbonate or bicarbonate</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000653</td>
<td><a href="../biochem_oxygen_dem/">biochem_oxygen_dem</a></td>
<td>0..1</td>
<td>Amount of dissolved oxygen needed by aerobic biological organisms in a body of water to break down organic material present in a given water sample at certain temperature over a specific time period</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000656</td>
<td><a href="../chem_oxygen_dem/">chem_oxygen_dem</a></td>
<td>0..1</td>
<td>A measure of the capacity of water to consume oxygen during the decomposition of organic matter and the oxidation of inorganic chemicals such as ammonia and nitrite</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000657</td>
<td><a href="../efficiency_percent/">efficiency_percent</a></td>
<td>0..1</td>
<td>Percentage of volatile solids removed from the anaerobic digestor</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000662</td>
<td><a href="../indust_eff_percent/">indust_eff_percent</a></td>
<td>0..1</td>
<td>Percentage of industrial effluents received by wastewater treatment plant</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000425</td>
<td><a href="../nitrate/">nitrate</a></td>
<td>0..1</td>
<td>Concentration of nitrate in the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000505</td>
<td><a href="../phosphate/">phosphate</a></td>
<td>0..1</td>
<td>Concentration of phosphate</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000183</td>
<td><a href="../salinity/">salinity</a></td>
<td>0..1</td>
<td>The total concentration of all dissolved salts in a liquid or solid sample. While salinity can be measured by a complete chemical analysis, this method is difficult and time consuming. More often, it is instead derived from the conductivity measurement. This is known as practical salinity. These derivations compare the specific conductance of the sample to a salinity standard such as seawater.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000110</td>
<td><a href="../samp_store_temp/">samp_store_temp</a></td>
<td>0..1</td>
<td>Temperature at which sample was stored, e.g. -80 degree Celsius</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000669</td>
<td><a href="../sludge_retent_time/">sludge_retent_time</a></td>
<td>0..1</td>
<td>The time activated sludge remains in reactor</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000428</td>
<td><a href="../sodium/">sodium</a></td>
<td>0..1</td>
<td>Sodium concentration in the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000102</td>
<td><a href="../tot_nitro/">tot_nitro</a></td>
<td>0..1</td>
<td>Total nitrogen concentration of water samples, calculated by: total nitrogen = total dissolved nitrogen + particulate nitrogen. Can also be measured without filtering, reported as nitrogen</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000689</td>
<td><a href="../tot_phosphate/">tot_phosphate</a></td>
<td>0..1</td>
<td>Total amount or concentration of phosphate</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000751</td>
<td><a href="../chem_administration/">chem_administration</a></td>
<td>0..*</td>
<td>List of chemical compounds administered to the host or site where sampling occurred, and when (e.g. Antibiotics, n fertilizer, air filter); can include multiple compounds. For chemical entities of biological interest ontology (chebi) (v 163), http://purl.bioontology.org/ontology/chebi</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000348</td>
<td><a href="../pre_treatment/">pre_treatment</a></td>
<td>0..1</td>
<td>The process of pre-treatment removes materials that can be easily collected from the raw wastewater</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000349</td>
<td><a href="../primary_treatment/">primary_treatment</a></td>
<td>0..1</td>
<td>The process to produce both a generally homogeneous liquid capable of being treated biologically and a sludge that can be separately treated or processed</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000350</td>
<td><a href="../reactor_type/">reactor_type</a></td>
<td>0..1</td>
<td>Anaerobic digesters can be designed and engineered to operate using a number of different process configurations, as batch or continuous, mesophilic, high solid or low solid, and single stage or multistage</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000755</td>
<td><a href="../samp_store_loc/">samp_store_loc</a></td>
<td>0..1</td>
<td>Location at which sample was stored, usually name of a specific freezer/room</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000351</td>
<td><a href="../secondary_treatment/">secondary_treatment</a></td>
<td>0..1</td>
<td>The process for substantially degrading the biological content of the sewage</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000215</td>
<td><a href="../sewage_type/">sewage_type</a></td>
<td>0..1</td>
<td>Type of wastewater treatment plant as municipial or industrial</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000352</td>
<td><a href="../tertiary_treatment/">tertiary_treatment</a></td>
<td>0..1</td>
<td>The process providing a final treatment stage to raise the effluent quality before it is discharged to the receiving environment</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000353</td>
<td><a href="../wastewater_type/">wastewater_type</a></td>
<td>0..1</td>
<td>The origin of wastewater such as human waste, rainfall, storm drains, etc.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000660</td>
<td><a href="../emulsions/">emulsions</a></td>
<td>0..*</td>
<td>Amount or concentration of substances such as paints, adhesives, mayonnaise, hair colorants, emulsified oils, etc.; can include multiple emulsion types</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000661</td>
<td><a href="../gaseous_substances/">gaseous_substances</a></td>
<td>0..*</td>
<td>Amount or concentration of substances such as hydrogen sulfide, carbon dioxide, methane, etc.; can include multiple substances</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000664</td>
<td><a href="../inorg_particles/">inorg_particles</a></td>
<td>0..*</td>
<td>Concentration of particles such as sand, grit, metal particles, ceramics, etc.; can include multiple particles</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000752</td>
<td><a href="../misc_param/">misc_param</a></td>
<td>0..*</td>
<td>Any other measurement performed or parameter collected, that is not listed here</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000665</td>
<td><a href="../org_particles/">org_particles</a></td>
<td>0..*</td>
<td>Concentration of particles such as faeces, hairs, food, vomit, paper fibers, plant material, humus, etc.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000672</td>
<td><a href="../soluble_inorg_mat/">soluble_inorg_mat</a></td>
<td>0..*</td>
<td>Concentration of substances such as ammonia, road-salt, sea-salt, cyanide, hydrogen sulfide, thiocyanates, thiosulfates, etc.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000673</td>
<td><a href="../soluble_org_mat/">soluble_org_mat</a></td>
<td>0..*</td>
<td>Concentration of substances such as urea, fruit sugars, soluble proteins, drugs, pharmaceuticals, etc.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000150</td>
<td><a href="../suspend_solids/">suspend_solids</a></td>
<td>0..*</td>
<td>Concentration of substances including a wide variety of material, such as silt, decaying plant and animal matter; can include multiple substances</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000103</td>
<td><a href="../organism_count/">organism_count</a></td>
<td>0..*</td>
<td>Total cell count of any organism (or group of organisms) per gram, volume or area of sample, should include name of organism followed by count. The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr).</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000754</td>
<td><a href="../perturbation/">perturbation</a></td>
<td>0..*</td>
<td>Type of perturbation, e.g. chemical administration, physical disturbance, etc., coupled with perturbation regimen including how many times the perturbation was repeated, how long each perturbation lasted, and the start and end time of the entire perturbation period; can include multiple perturbation types</td>
<td>None</td>
<td>.</td>
</tr>
</tbody>
</table>
<h2 id="usages">Usages</h2>
<h2 id="identifier-and-mapping-information">Identifier and Mapping Information</h2>

              
            </article>
            
          </div>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    <script id="__config" type="application/json">{"base": "..", "features": [], "search": "../assets/javascripts/workers/search.b97dbffb.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.6c7ad80a.min.js"></script>
      
    
  </body>
</html>